core-bundle/contao/templates/backend/be_tinyMCE.html5

```diff
@@ -16,26 +16,28 @@ window.tinymce && tinymce.init({
   document_base_url: '<?= Environment::get('base') ?>',
   entities: '160,nbsp,60,lt,62,gt,173,shy',
   branding: false,
+  promotion: false,
   skin: 'tinymce-5',
   setup: function(editor) {
     editor.getElement().removeAttribute('required');
+    document.querySelectorAll('[accesskey]').forEach(function(el) {
+      editor.addShortcut('access+' + el.accessKey, el.id, () => el.click());
+    });
   },
   init_instance_callback: function(editor) {
     if (document.activeElement && document.activeElement.id && document.activeElement.id == editor.id) {
       editor.editorManager.get(editor.id).focus();
     }
-    editor.on('focus', function() { Backend.getScrollOffset(); });
+    editor.on('focus', () => Backend.getScrollOffset());
```

Comment:
this is still an arrow function

Reply:
That's what the commit message says, no?
![image](https://user-images.githubusercontent.com/5305677/189854536-bb6db0a2-159c-4a0c-9496-56d6d86bbf30.png)

Reply:
See https://github.com/contao/contao/pull/5266#discussion_r968344179